input.go

```diff
@@ -52,13 +52,13 @@ func IsKeyPressed(key Key) bool {
 	var keys []driver.Key
 	switch key {
 	case KeyAlt:
-		keys = append(keys, driver.KeyLeftAlt, driver.KeyRightAlt)
+		keys = []driver.Key{driver.KeyLeftAlt, driver.KeyRightAlt}
 	case KeyControl:
-		keys = append(keys, driver.KeyLeftControl, driver.KeyRightControl)
+		keys = []driver.Key{driver.KeyLeftControl, driver.KeyRightControl}
 	case KeyShift:
-		keys = append(keys, driver.KeyLeftShift, driver.KeyRightShift)
+		keys = []driver.Key{driver.KeyLeftShift, driver.KeyRightShift}
 	default:
-		keys = append(keys, driver.Key(key))
+		keys = []driver.Key{driver.Key(key)}
```

Comment:
Just out of curiosity, I was wondering why this reduced heap allocation. What was the difference? @corfe83

Reply:
NVM: From https://go.godbolt.org/z/3gbhBz and https://go.godbolt.org/z/9w2ptQ, slice values are on the stack.